### PR TITLE
Reduce resync

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#1902

Reduces resync period from 5 mins to 1 min.  Also cancels whole reconciliation when the guest cluster is not up yet and there are following resources that access it.

This is to speed up how quickly we can install chartconfigs so chart-operator can do its thing. The problem is when the chartconfig CRD becomes available. The chart resource installs chart-operator but the CRD is only created when the operator boots.

So we need to wait up until 5 mins for the next resync before the chartconfig resource can create CRs.

### CODE REVIEW

File: pkg/v3/resource/namespace/current.go [hunk 1]
```diff
@@ -33,8 +34,8 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 
 		// We can't continue without a K8s client. We will retry during the
 		// next execution.
-		resourcecanceledcontext.SetCanceled(ctx)
-		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource reconciliation for custom object")
+		reconciliationcanceledcontext.SetCanceled(ctx)
+		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling reconciliation for custom object")
```
Comment (rossf7) [inline pkg/v3/resource/namespace/current.go]: Same applies for the namespace resource which is executed before the chart and chartconfig resources.

**Comment by rossf7** [comment]
@tuommaki FYI and let me know if you see any problems reducing the resync period to 1 min.

**Comment by tuommaki** [comment]
> let me know if you see any problems reducing the resync period to 1 min.

I don't think there should be any problems with this. :+1:

**Comment by rossf7** [comment]
@tuommaki Thx for the confirmation👍

**Pull request merged**